src/main/java/hudson/remoting/Channel.java

```diff
@@ -1716,6 +1718,24 @@ public static Channel current() {
         return CURRENT.get();
```

Comment:
Should this method be deprecated, or maybe update the JavaDoc to indicate that `Channel#currentOrIllegalState` or `SerializableOnlyOverRemoting#getChannelForSerDes` should probably be used instead?

Reply:
The method itself is fine, API users just need to use it correctly. I would keep it